src/elements/FormTextInput/FormTextInput.tsx

```diff
@@ -10,6 +10,7 @@ import styles from './FormTextInput.styles';
 
 interface FormTextInputProps {
 	text: string;
+	placeholder: string;
```

Comment:
Add a question mark at the end of the property name, like 'width'. This makes the property optional within the interface